ui/org.eclipse.pde.launching/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %name
 Bundle-SymbolicName: org.eclipse.pde.launching;singleton:=true
-Bundle-Version: 3.13.400.qualifier
+Bundle-Version: 3.14.0.qualifier
```

Comment:
The API-tools currently say this would be necessary, but in the current state I don't see a reason, because only package-private methods are added.
But let's see if we these methods become `protected` eventually.

Reply:
As far as I can tell, API-tools considers the removal of the overridden `EquinoxLaunchConfiguration.preLaunchCheck()` method falsely as an addition. I have now suppressed this error.